runtime/src/main/java/com/flipkart/flux/resource/FluxUIResource.java

```diff
@@ -0,0 +1,105 @@
+package com.flipkart.flux.resource;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.flipkart.flux.FluxRunTimeService;
+import com.flipkart.flux.commons.dto.WorkFlowStateSummary;
+import com.flipkart.flux.commons.dto.WorkFlowStatesDetail;
+import com.flipkart.flux.commons.dto.WorkflowSummary;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import javax.inject.Named;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/api/teams")
+@Named
+@Singleton
+public class FluxUIResource {
```

Comment:
Fair point. I am marking it todo right now. Have to discuss about StateMachine class a bit. Changed the name to FluxResource though.